### PR TITLE
Bump ci ember-cli versions to 3.28

### PR DESCRIPTION
**Pull request by abeforgit**
Bump ci ember-cli versions

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -4,24 +4,24 @@ type: docker
 name: test
 steps:
 - name: install
-  image: danlynn/ember-cli:3.20.0
+  image: danlynn/ember-cli:3.28.5
```
Comment (nvdk) [inline .drone.yml:7]:
```suggestion
  image: danlynn/ember-cli:3.28.5-node_16.13
```
would suggest bumping to node 16 here as well

Comment (abeforgit) [reply .drone.yml:7]: but 3.28.5 already uses node 16

Comment (abeforgit) [reply .drone.yml:7]: https://github.com/danlynn/ember-cli/blob/3.28.5/Dockerfile

Comment (abeforgit) [reply .drone.yml:7]: the tags are identical

Comment (abeforgit) [reply .drone.yml:7]: what the hell I made this pr months ago haha

**Pull request closed without merging**